### PR TITLE
Switch to Colors package (do not merge yet)

### PR DESCRIPTION
**Pull request by timholy**
Do not merge until C-day (hopefully later this week).

I'm not certain how you test this. The example [here](https://github.com/stevengj/PyPlot.jl#basic-usage) seems to work fine, from what I can tell.

### CODE REVIEW

File: src/colormaps.jl [hunk 1]
```diff
@@ -79,28 +79,28 @@ function ColorMap{T<:Real}(name::Union(AbstractString,Symbol),
            name, segmentdata, n, gamma)
 end
 
-typealias AColorValue Union(ColorValue,AbstractAlphaColorValue)
+typealias AColor Union(Color,TransparentColor)
```
Comment (stevengj) [inline src/colormaps.jl]: It seems like `AColor` can go away and be replaced by `Colorant`

File: src/colormaps.jl [hunk 2]
```diff
@@ -79,28 +79,28 @@ function ColorMap{T<:Real}(name::Union(AbstractString,Symbol),
...
 
 # create from an array c, assuming linear mapping from [0,1] to c
-function ColorMap{T<:AColorValue}(name::Union(AbstractString,Symbol),
+function ColorMap{T<:AColor}(name::Union(AbstractString,Symbol),
                                   c::AbstractVector{T},
                                   n=max(256, length(c)), gamma=1.0)
     nc = length(c)
     if nc == 0
-        throw(ArgumentError("ColorMap requires a non-empty ColorValue array"))
+        throw(ArgumentError("ColorMap requires a non-empty Color array"))
```
Comment (stevengj) [inline src/colormaps.jl]: Colorant

**Comment by stevengj** [comment]
(Yes, I wish I had automated tests for this package, but am not sure how to go about it.)

**Comment by timholy** [comment]
Winston and Compose export to a PDF or similar and compare the results against a cached version. I did have to add some exception logic to keep the tests from being too finicky, though: https://github.com/dcjones/Compose.jl/commit/18312ff057b7305216ea38c93fd9cb4cdcb6be80

Fixes for your review comments are on the way.

**Comment by timholy** [comment]
OK, all set. I checked that I didn't break the README example (on both 0.3 and 0.4).

**Comment by timholy** [comment]
I should also add that Winston has seem some occasional annoyance, e.g., https://github.com/nolta/Winston.jl/pull/213#issuecomment-101358131

**Comment by timholy** [comment]
Combined testing here: https://travis-ci.org/timholy/Dummy.jl. Assuming this is on-track tomorrow, I'll merge.

**Comment by stevengj** [comment]
Great, thanks!

**Comment by timholy** [comment]
Merge away.

**Pull request merged**